.github/workflows/ci.yml

```diff
@@ -53,17 +53,19 @@ jobs:
     - uses: actions/checkout@v2.3.4
     - name: Install project dependencies
       run: |
-        brew list cmake || brew cask install cmake
-        brew list gfortran || brew cask install gfortran
+        echo "Trying to find latest gfortran version in /usr/local/Cellar/gcc"
```

Comment:
This is very specific. Is `which` or `whence` appropriate to find out if `gfortran` is available as a command instead?  I mean, do we really care if it is in these areas, or is it OK to be elsewhere, such as `/usr/bin/gfortran`?

Reply:
I am surprised by the Mac system but I think I tried `/usr/bin/gfortran` which did not work. The current way was the best I could think of and that actually works. 

Do you know if it is possible to "debug" the build process in github actions ? I agree that there may be better ways to find the gfortran compiler but "which gfortran" did not work, so this was the second best solution. See also https://github.com/exercism/fortran/pull/126

Reply:
Can these CI/CD changes be removed from the pull request after #126 ?

Reply:
I don't know… I am currently in the process of learning more about github actions myself.  But I just saved this link so that if I come across something, I can get back to you.

Reply:
I am talking about locating that, or verifying that it is in the path, though.  `which gfortran` should give a positive answer if it is found in the path.  This is more important I think than the specific location.  Or less brittle should the installation location change.

Reply:
>Can these CI/CD changes be removed from the pull request after #126 ?
Probably but I am too bad at git to make that happen. I would rather suggest to leave the CI/CD stuff for now and clean that up later, if really necessary. It works now and I think it is really hard to debug the CI-stuff.

Reply:
I was really hoping to hear some good tips on debugging the CI stuff.

Reply:
I'll create an issue to revisit & simplify CI/CD

I've seen it getting somewhat major changes across many pull requests and I can't tell what are the same changes or slightly different ones or if they need to get resolved in a certain order etc...

But I think the problem is just that the PRs aren't trying to focus on it. The PRs are interested in adding exercises (and they should remain focused on that) We can just say "Let's come back to it later" and give the CI/CD some proper focus